OpenRA.Mods.Common/Traits/World/Pathfinder/PathFinderInfo.cs

```diff
@@ -0,0 +1,51 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Calculates routes for mobile units based on the A* search algorithm.", " Attach this to the world actor.")]
+	public class PathFinderInfo : ITraitInfo
```

Comment:
Since the cache is kind of an extension and a supportish feature to the main trait, I'll put it in the pathfinder folder and namespace. We can always change it if we decide otherwise.